src/resolve-info.ts

```diff
@@ -172,12 +174,24 @@ type FragmentsType = GraphQLResolveInfo["fragments"];
 type GraphQLNamedOutputType = GraphQLNamedType & GraphQLOutputType;
 type GraphQLObjectLike = GraphQLInterfaceType | GraphQLObjectType;
 
-const memoizedGetReturnType = memoize2(getReturnType);
-const memoizedHasField = memoize2(hasField);
-const memoizedResolveEndType = memoize(resolveEndType);
-const memoizedGetPossibleTypes = memoize2(getPossibleTypes);
-const memoizedExpandFieldNodeType = memoize4(expandFieldNodeType);
-const memoizedExpandFieldNode = memoize4(expandFieldNode);
+const MEMOIZATION = true;
```

Comment:
Could you explain why we need that constant?

Reply:
Only for debugging purposes when changing / updating the implementation of field filtering.